btsieve/src/lib.rs

```diff
@@ -38,3 +40,27 @@ impl PartialOrd for QueryMatch {
         self.0.partial_cmp(&other.0)
     }
 }
+
+struct BlockchainDAG<T, V> {
```

Comment:
The Blockchain DAG is not needed for this PR :)

Reply:
I know, but otherwise this would have been too easy :D